@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -380,30 +392,111 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
                 });
             }
 
-            const localLine: number[] = [];
-            const worldLine: number[] = [];
-            const lineUvs: number[] = [];
-            const lineOffsets: number[] = [];
-            polyline.positions.forEach(pos => {
-                webMercatorTile2TargetWorld(extents, this.m_decodeInfo, pos, tmpV3);
-                worldLine.push(tmpV3.x, tmpV3.y, tmpV3.z);
-
-                if (computeTexCoords) {
-                    const { u, v } = computeTexCoords(pos, extents);
-                    lineUvs.push(u, v);
+            // Add continuous line as individual segments to improve special dashes by overlapping
+            // their connecting vertices. The technique/style should defined round or rectangular
+            // caps.
+            if (hasIndividualLineSegments) {
+                localLineSegments = [];
+
+                // Compute length of whole line and offsets of individual segments.
+                let lineLength = 0;
+                if (polyline.positions.length > 1) {
+                    const pointCount = polyline.positions.length;
```

Comment:
Move `pointCount` two line above so we may use it in `if` condition.

Reply:
Done.